### ISSUE TITLE
Add Brightness increase and descrease options

### ISSUE THREAD

**Issue by bloojoop** [FalconChristmas/fpp-brightness#3]
If not too much trouble, it would be nice to have a fpp command included that would increase and decrease the brightness.  Up and down by 10 %  each way would be good.  A use case I am thinking is using the big buttons plugin to have a brightness up and brightness down option.

**Comment by patdelaney** [comment]
Did you look at the brightness plugin, will that do what you want?

**Comment by bloojoop** [comment]
I spoke with Pat on this request this evening to explain that this request for for additional command option gets added to the brightness plug in.  A way to incrementally increase or decrease the brightness value.  During our conversation I realized that maybe I could write a script to do this.  something like : 
retrieve current brightness and store it
store current brightness + 10 (for an increase -10 for decrease)
set brightness to new value
Then my big button can just call the script instead of an fpp command.  I will look into this method.

**Comment by bloojoop** [comment]
quick update.  The script method seems to work out.  I have attached my rough scripts.  I noticed that on recent builds (since moving away from master/standalone) that the brightness plugin does not send to remotes (see issue 4).  I also noticed when configuring a big button with the big buttons plugin, it does not allow you to set multicast on commands.  I also got around this by configuring a command preset to run the script with multicast enabled, then configured the button to trigger the command preset.  I configured a script for up and down and a command set for up and down and a button for up and down.  I seem to have the effect I was looking for.  I will say this method add some delay between hitting the button and the brightness changing. 

[up.txt](https://github.com/FalconChristmas/fpp-brightness/files/6980318/up.txt)
[down.txt](https://github.com/FalconChristmas/fpp-brightness/files/6980320/down.txt)
